examples/pytorch/ogb/deepwalk/deepwalk.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, args):
             fast_neg=args.fast_neg,
             ogbl_name=args.ogbl_name,
             load_from_ogbl=args.load_from_ogbl,
+            ogbn_name=args.ogbn_name,
```

Comment:
It seems the logic of loading data from ogbn is similar to that from ogbl. How about using `load_from_ogb` and `ogb_name` to unify the entrance and branching in `load_from_ogb_with_name` function according to the prefix of dataset name?